packages/platforms/browser/lib/auto-instrumentation/page-load-phase-spans.ts

```diff
@@ -1,36 +1,49 @@
-import type { SpanFactory } from '@bugsnag/core-performance'
+import type { SpanContext, SpanFactory } from '@bugsnag/core-performance'
 import { type PerformanceWithTiming } from '../on-settle/load-event-end-settler'
 
 function shouldOmitSpan (startTime?: number, endTime?: number): boolean {
   return (startTime === undefined || endTime === undefined) ||
   (startTime === 0 && endTime === 0)
 }
 
-export const instrumentPageLoadPhaseSpans = (spanFactory: SpanFactory, route: string, performance: PerformanceWithTiming) => {
+export const instrumentPageLoadPhaseSpans = (
+  spanFactory: SpanFactory,
+  performance: PerformanceWithTiming,
+  route: string,
+  pageLoadSpan: SpanContext
+) => {
   const entry = performance.getEntriesByType('navigation')[0] as PerformanceNavigationTiming
 
   if (entry) {
     if (!shouldOmitSpan(entry.unloadEventStart, entry.unloadEventEnd)) {
       spanFactory.endSpan(spanFactory.startSpan('[PageLoadPhase/Unload]' + route, {
-        startTime: entry.unloadEventStart
+        startTime: entry.unloadEventStart,
+        parentContext: pageLoadSpan,
+        makeCurrentContext: false
       }), entry.unloadEventEnd)
     }
```

Comment:
there's actually quite a lot of repeated code here, which could probably be abstracted into a helper function which accepts a name, start/end time and the span factory

Reply:
good shout! updated